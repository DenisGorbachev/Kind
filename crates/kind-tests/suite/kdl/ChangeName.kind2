#kdl_name = JOJO
#keep
Jonathan.Joestar : U60
Jonathan.Joestar = 42