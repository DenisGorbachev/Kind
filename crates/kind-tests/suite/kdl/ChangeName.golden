fun (JOJO ) {
  (JOJO) = #42
}

